docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -196,19 +205,19 @@ distributed training:
    distributed training jobs are not scheduled and wait indefinitely. For example, if every agent in
    the cluster has eight GPUs, a job with ``slots_per_trial`` set to ``12`` is never scheduled.
 
-   If a multi-GPU experiment does not become active after a minute or so, please confirm that
-   ``slots_per_trial`` is set so that it can be scheduled within these constraints. The CLI command
-   ``det task list`` can also be used to check if any other tasks are using GPUs and preventing your
+   If a multi-GPU experiment does not become active after a minute or so, please ensure that
+   ``slots_per_trial`` is set so that it can be scheduled within these constraints. You can also use
+   the CLI command ``det task list`` to check if any other tasks are using GPUs and preventing your
```

Comment:
Redundant, again.

Reply:
This should be resolved now that the info is consolidated in the Scheduling Behavior section.